addOns/soap/src/main/java/org/zaproxy/zap/extension/soap/WSDLCustomParser.java

```diff
@@ -202,7 +204,13 @@ private void parseWSDLUrl(String url) {
                 }
             }
             /* Sends a request to retrieve remote WSDL file's content. */
-            HttpMessage httpRequest = new HttpMessage(new URI(url, true));
+            HttpMessage httpRequest = null;
+            try {
+                httpRequest = new HttpMessage(new URI(url, true));
+            } catch (URIException ue) {
+                LOG.warn("Invalid URL: {} . {}", url, ue.getMessage());
```

Comment:
This fits in with the current code, but ideally these messages should be passed 'up' so that they can be displayed to the user in the GUI (or reported as errors via the AF etc) rather than the user having to look at the zap.log file.

Reply:
It can probably be re-thrown and a warning dialog shown. I'll look at that.

Reply:
It seems this class was already popping dialogs for other failure conditions. I know it's not good form to mix processing and UI, but I followed suit.